### PR TITLE
[Controller] Use common.GetLabels instead of mariadb.GetLabels

### PR DESCRIPTION
**Pull request by zerodayz**
Use common.GetLabels instead of mariadb.GetLabels.

### CODE REVIEW

File: controllers/mariadb_controller.go [hunk 1]
```diff
@@ -225,8 +225,7 @@ func (r *MariaDBReconciler) generateServiceConfigMaps(
 	instance *databasev1beta1.MariaDB,
 	envVars *map[string]common.EnvSetter,
 ) error {
-	// FIXME: use common.GetLabels?
-	cmLabels := mariadb.GetLabels(instance.Name)
+	cmLabels := common.GetLabels(instance, mariadb.AppLabel, map[string]string{})
```
Comment (abays) [inline controllers/mariadb_controller.go:228]: We've actually reorganized the `lib-common` repo in the past few weeks.  An example of how we use the `labels` package can be seen here:

(import)
https://github.com/openstack-k8s-operators/keystone-operator/blob/0dc6665402274f87a648d1e6305c4c67b783b857/controllers/keystoneapi_controller.go#L35

(use)
https://github.com/openstack-k8s-operators/keystone-operator/blob/0dc6665402274f87a648d1e6305c4c67b783b857/controllers/keystoneapi_controller.go#L620

It looks like this operator needs a general refactor to `go get -u github.com/openstack-k8s-operators/lib-common/module/common` and then rework the existing code to use the new `lib-common` approach.

Comment (zerodayz) [reply controllers/mariadb_controller.go:228]: Is something wrong with the lib-common package? When I use package `github.com/openstack-k8s-operators/lib-common` instead it shows v0.0.0 so I am not sure whether anything changed ....

```
mariadb-operator $ go get -u github.com/openstack-k8s-operators/lib-common/module/common
go: module github.com/openstack-k8s-operators/lib-common@upgrade found (v0.0.0), but does not contain package github.com/openstack-k8s-operators/lib-common/module/common
```

**Comment by zerodayz** [comment]
I have also tested locally and mariadb still worked.

```
oc get pods -w
NAME                      READY   STATUS    RESTARTS   AGE
openstack-db-init-wblxb   0/1     Pending   0          0s
openstack-db-init-wblxb   0/1     Pending   0          2s
openstack-db-init-wblxb   0/1     ContainerCreating   0          2s
openstack-db-init-wblxb   0/1     ContainerCreating   0          3s
openstack-db-init-wblxb   1/1     Running             0          7s
openstack-db-init-wblxb   0/1     Completed           0          18s
openstack-db-init-wblxb   0/1     Completed           0          20s
openstack-db-init-wblxb   0/1     Terminating         0          25s
openstack-db-init-wblxb   0/1     Terminating         0          26s
mariadb-openstack         0/1     Pending             0          0s
mariadb-openstack         0/1     Pending             0          0s
mariadb-openstack         0/1     ContainerCreating   0          0s
mariadb-openstack         0/1     ContainerCreating   0          2s
mariadb-openstack         1/1     Running             0          6s
```

**Review by abays — APPROVED** [review]
/lgtm

**Pull request merged**